### PR TITLE
Issue/#288 Explicitly set game start time

### LINKED ISSUE #288: Explicitly set startTime
The server needs to set `game_stats.startTime` explicitly. See FAForever/db#83

### PR DESCRIPTION
**Pull request by KaukaHan**
closes #288

### CODE REVIEW

File: server/games/game.py [hunk 1]
```diff
@@ -758,6 +758,7 @@ async def update_game_stats(self):
                     mapId=self.map_id,
                     gameName=self.name,
                     validity=self.validity.value,
```
Comment (Askaholic) [inline server/games/game.py:760]: I think the point of the issue is that the collection of the start time and the database insert should be separated. As in it should be possible to move the database insert to an earlier or later time and the start time should still be recorded correctly.

**Comment by Askaholic** [comment]
You should get rid of the unrelated commits. Let me know if you need some help with the git commands to do this

**Comment by KaukaHan** [comment]
> You should get rid of the unrelated commits. Let me know if you need some help with the git commands to do this
that would be helpfull

File: server/games/game.py [hunk 2]
```diff
@@ -80,6 +81,8 @@ def __init__(
         self.game_service = game_service
         self._player_options: Dict[int, Dict[str, Any]] = defaultdict(dict)
         self.launched_at = None
+        self.startTime = None
+        self.endTime = None
```
Comment (Askaholic) [inline server/games/game.py:85]: Variable names should be snake_case

Comment (Askaholic) [reply server/games/game.py:85]: I know the table columns aren’t, but that’s just cuz the database schema is a bit of a mess

**Review by Askaholic — COMMENTED** [review]
Looks good. Now we just need some unit tests to check that the variables are set correctly and maybe one to check that the right value is written to the database.

File: server/games/game.py (hunk 2 above)
Comment (KaukaHan) [reply server/games/game.py:85]: do i have to use hardcoded timestemps for the tests ?

Comment (Askaholic) [reply server/games/game.py:85]: You should probably mock the calls to `datetime.now` to return a specific value and then assert that the value shows up in the database.

File: server/games/game.py [hunk 3]
```diff
@@ -416,12 +419,13 @@ async def check_sim_end(self):
         if [conn for conn in self.connections if not conn.finished_sim]:
             return
         self.ended = True
+        self.endTime = datetime.datetime.now(datetime.timezone.utc)
```
Comment (Askaholic) [inline server/games/game.py:422]: I'd probably import like:
```python3
from datetime import datetime, timezone
```
Just so these calls are a little cleaner.

**Comment by Askaholic** [comment]
No work happening on this

**Pull request closed without merging**